integrationtest/cli/cli_test.go

```diff
@@ -196,6 +203,20 @@ func TestIntegration(t *testing.T) {
 			status:  cmd.ExitCodeOK,
 			stdout:  fmt.Sprintf("%s (aws_instance_example_type)", color.New(color.Bold).Sprint("instance type is t2.micro")),
 		},
+		{
+			name:    "`--minimum-severity` option with issues and severity level info",
+			command: "./tflint --minimum-severity=info",
+			dir:     "issues_found",
+			status:  cmd.ExitCodeIssuesFound,
+			stdout:  fmt.Sprintf("%s (aws_instance_example_type)", color.New(color.Bold).Sprint("instance type is t2.micro")),
+		},
+		{
+			name:    "`--minimum-severity` option with issues and severity level warning",
+			command: "./tflint --minimum-severity=warning",
+			dir:     "issues_found",
+			status:  cmd.ExitCodeOK,
+			stdout:  fmt.Sprintf("%s (aws_instance_example_type)", color.New(color.Bold).Sprint("instance type is t2.micro")),
```

Comment:
Since the `aws_instance_example_type` severity is error, it would be wrong for this to return ExitCodeOK.
You can use the following rules in the integration tests:
https://github.com/terraform-linters/tflint/tree/7b4785d2a7515d536c8faf2bb16096829414b1dd/plugin/stub-generator/sources/testing/rules

Reply:
Added more and improved tests 👍